src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -12,8 +12,6 @@
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Sql</PackageId>
     <PackageTags>Microsoft Azure WebJobs AzureFunctions SQL AzureSQL</PackageTags>
```

Comment:
You should probably add package tags to the worker extension too. See if there's any other tags that other work extensions use (in addition to these ones)

They can stay in the separate csproj's though since those are like IDs in that they are attributed more directly to the specific package being generated.